package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "spike",
-  "version": "1.0.0",
+  "name": "espina",
+  "version": "1.0.2",
   "description": "Starter kit and generator for ES6 React Isomorphic webapp (React, Redux, Webpack, Universal-Webpack, Karma, Jasmine)",
-  "main": "run.js",
+  "main": "./dist/main.js",
   "scripts": {
-    "test": "BABEL_DISABLE_CACHE=1 babel-node test.js"
+    "test": "BABEL_DISABLE_CACHE=1 babel-node test.js",
+    "build": "babel src/espina --out-dir . --ignore mock.js,test.js,test.html",
```

Comment:
I can't create a single file with all the classes using babel, maybe we need other compiler or something